docs/whats-new.asciidoc

```diff
@@ -1,78 +1,113 @@
+[discrete]
+=== Wilcard support for event filters
+
+{security-guide}/event-filters.html[Event filters] now support using wildcard entries for the `file.path.text` field using the `matches` operator.
 
-To view network map data, you now need at least `Read` privileges for `Maps`. Refer to {security-guide}/conf-map-ui.html[Configure network map data] for more information.
+[discrete]
+=== Detection rules enhancements
 
-*Alert enhancements*
+*Rule execution logs*
 
-On the Alerts page, the Trend and Count tables now allow you to aggregate alert data based on any aggregatable ECS field. Previously, only a handful of fields were visible.
+The new {security-guide}/alerts-ui-monitor.html#rule-execution-logs[*Rule execution logs*] tab on a rule's details page provides historical data for the rule's executions over time. Use this to understand how a particular rule is running and whether it’s creating the alerts you expect
 
 [role="screenshot"]
-image::whats-new/images/8.1/counts-table.gif[]
+image::whats-new/images/8.2/rule-exec-logs.png[]
+
+*Bulk apply a Timeline template to rules*
+
+A new {security-guide}/rules-ui-management.html#edit-rules-settings[bulk actions option] allows you to apply a Timeline template to multiple rules at once.
+
+*New Rules table filter options*
 
-The {security-guide}/alerts-ui-manage.html#view-alert-details[alert details flyout] has a new layout that highlights relevant fields. You can also now update an alert's status directly from the flyout.
+You can now {security-guide}/rules-ui-management.html#sort-filter-rules[filter the Rules table] by index pattern, MITRE ATT&CK tactic, technique (name or ID), and rule name.
+
+*Rule preview feature includes alerts table*
+
+When you create or edit a detection rule and preview it, the {security-guide}/rules-ui-create.html#preview-rules[rule preview] now includes an alerts table with the expected alerts for the rule. Use this feature to learn how noisy a rule may be before saving it. You can now also preview {security-guide}/rules-ui-create.html#create-indicator-rule[indicator match rules].
 
 [role="screenshot"]
-image::whats-new/images/8.1/alert-details.gif[]
+image::whats-new/images/8.2/preview-rules.png[]
 
-*New Cases metrics*
+*Turn off `read` privilege warnings for detection rules*
 
-{security-guide}/cases-overview.html[Cases] now have metrics that summarize alert information and response times.
+A new Advanced Settings toggle, `securitySolution:enableCcsWarning`, allows you to {security-guide}/advanced-settings.html#enable-ccs-warning[turn off `read` privilege warnings] for detection rules using a remote cross-cluster search (CCS) index pattern.
+
+[discrete]
+=== Alert details enhancements
+
+You can now {security-guide}/alerts-run-osquery.html[run Osquery searches] from the Take action menu on the Alert details flyout.
 
 [role="screenshot"]
-image::whats-new/images/8.1/cases-kpis.png[]
+image::whats-new/images/8.2/run-osquery.png[]
```

Comment:
Should we single-source this image? Would that also help the resolution?

Reply:
I think the image is about as best as we can do here.